client/src/routes/Menu/index.jsx

```diff
@@ -98,6 +105,9 @@ function Menu() {
           <SemiCircle title="" className={semiCircle} />
```

Comment:
Title attribute should be removed.

Reply:
Title attribute removed. 

I increased the z-index of the logo and move the circle slightly to the right. 

I made adjustments to the logo and main image position so they wouldn't overlap on any screen.